src/views/Main/Catalog/Components/PermissionsPanel.js

```diff
@@ -101,10 +101,14 @@ export class PermissionsPanel extends React.Component {
 
   loadAcls = (updateSuccess) => {
     if (this.props.propertyTypeName) {
-      // PermissionsApi.getOwnerAclsForPropertyTypesInEntitySet(this.props.entitySetName)
-      // .then((acls) => {
-      //   console.log(acls);
-      // })
+      const fqn = {
```

Comment:
if you want, but I don't recommend it. I've been thinking about whether or not I need to add data model classes to help users of the api. each function takes in raw object literals right now, but it could be better if it was like...

```
SomeRequest req = new SomeRequest()
  .setNamespace(...)
  .setName(...)
  .setType(...)
  .setProperties(...)
  .build();

LoomApi.SomeApi.getBlahBlah(req)
```

... just trying to figure out if it makes sense to put the effort in to expose a ton of these data model classes, or to just leave it as is and document the raw object literals very well.